src/components/list/list.tsx

```diff
@@ -295,7 +295,7 @@ export class List {
     private triggerIconColorWarning() {
         if (this.items?.some((item) => 'iconColor' in item)) {
             console.warn(
-                "The `iconColor` prop is deprecated now! Use the new `Icon` interface and instead of `iconColor: 'color-name'` write `icon {name: 'icon-name', color: 'color-name'}`."
+                "The `iconColor` prop is deprecated, has no visual effect anymore, and will soon be removed! Use the new `Icon` interface and instead of `iconColor: 'color-name'` write `icon {name: 'icon-name', color: 'color-name'}`."
             );
         }
```

Comment:
cannot use the keyboard's Enter or Space keys, to open the menu, after using `limel-list-item`

<img width="497" height="277" alt="Image" src="https://github.com/user-attachments/assets/445de5cc-6bd3-4323-9c2e-96a9c4bbbeea" />